pkg/resources/ray/statefulset.go

```diff
@@ -318,7 +318,10 @@ func processArgs(rc *dcv1alpha1.RayCluster) []string {
 		"--num-cpus=$(MY_CPU_REQUEST)",
 		fmt.Sprintf("--object-manager-port=%d", rc.Spec.ObjectManagerPort),
 		fmt.Sprintf("--node-manager-port=%d", rc.Spec.NodeManagerPort),
-		fmt.Sprintf("--worker-port-list=%s", strings.Join(util.IntsToStrings(rc.Spec.WorkerPorts), ",")),
+	}
+
+	if rc.Spec.WorkerPorts != nil {
+		args = append(args, fmt.Sprintf("--worker-port-list=%s", strings.Join(util.IntsToStrings(rc.Spec.WorkerPorts), ",")))
```

Comment:
Is there a test for this new behavior?

Reply:
There is an existing test with worker ports set.  I didn't add a new test for empty worker ports given my limited ability in Go.  I will defer that to Sonny after he is back from vacation.

Reply:
@sonnysideup I recommend adding a new test case for this.

Notice that this PR decreased test coverage.

Reply:
The test coverage report looks iffy.

The report included 2 files that this PR did not modify where test coverage decreased:
- controllers/raycluster_controller.go (-2.47%)
- controllers/sparkcluster_controller.go (-1.34%)

A few lines got removed in this file and the test coverage decreased slightly, which is understandable:
- api/v1alpha1/raycluster_webhook.go (-0.10%)

A couple of new lines got added to this file and the test coverage remained at 100%:
- pkg/resources/ray/statefulset.go

Overall there is no net decrease in the number of lines covered in this PR.